installer/resources/pacbot_app/files/DB.sql

```diff
@@ -23,6 +23,52 @@ SET @region='$region';
 SET @account='$account';
 SET @eshost='$eshost';
 SET @esport='$esport';
+set @LOGGING_ES_HOST_NAME='$LOGGING_ES_HOST_NAME'
```

Comment:
@sujithgeorge 
Could you please use SET instead of set and please add semicolon at the end of each line
For ex: 
SET @LOGGING_ES_HOST_NAME='$LOGGING_ES_HOST_NAME';

This is required as we are using replacement algorithm in python to replace variable with real value.

Reply:
@sajeer-nooh  I have made the required changes. PR updated.

Reply:
Verified